benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/cassandra/CassandraTpchSchema.java

```diff
@@ -34,11 +34,10 @@
     primaryKeyMap.put("nation", new LinkedList<>(Arrays.asList("n_nationkey")));
```

Comment:
Shouldn't need to wrap these in a new list.

Reply:
If you could adjust this in all areas of the code.

Reply:
done